pkg/server/slicerouter_config.go

```diff
@@ -119,7 +120,32 @@ func vl3InjectRouteInKernel(dstIP string, nextHopIP string) error {
 	}
 	gwIP := net.ParseIP(nextHopIP)
 
-	route := netlink.Route{Dst: dstIPNet, Gw: gwIP}
+	installedRoutes, err := netlink.RouteList(nil, netlink.FAMILY_V4)
+	if err != nil {
+		return err
+	}
+
+	var linkIdx int = -1
+	for _, route := range installedRoutes {
+		logger.GlobalLogger.Errorf("Route: %v", route)
```

Comment:
@bharath-avesha why are these Error logs? shouldn't they be info logs?

Reply:
@rahulsawra98 yes, you are right. Those logs are not needed, was just using them during development. Updated the PR with those logs removed.